src/dss/utils.py

```diff
@@ -81,3 +113,62 @@ def get_lightkube_client(kubeconfig: Optional[str] = None):
     kubeconfig = KubeConfig.from_file(kubeconfig)
     lightkube_client = Client(config=kubeconfig)
     return lightkube_client
+
+
+def get_mlflow_tracking_uri() -> str:
```

Comment:
Just a thought we are checking if PVC exists but we dont check if mlflow exists. We are by default always sending this url. Maybe we should also check for MLflow.

Reply:
what should we do if mlflow does not exist? not create the notebook? or create the notebook without the `MLFLOW_TRACKING_URI` env var?

Reply:
I believe we dont expect a valid notebook without mlflow running ... I think we should not create notebook

Reply:
makes sense, I added the check alongside the PVC check